zeppelin-server/pom.xml

```diff
@@ -43,19 +43,19 @@
       <dependency>
         <groupId>org.scala-lang</groupId>
         <artifactId>scala-library</artifactId>
-        <version>2.10.4</version>
+        <version>2.11.8</version>
```

Comment:
@Leemoonsoo The selenium check is failing with the error below. Is this due to a known flaky test issue? cc @bzz

https://travis-ci.org/apache/incubator-zeppelin/jobs/135722600

Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 75.38 sec <<< FAILURE! - in org.apache.zeppelin.integration.ParagraphActionsIT
testRemoveButton(org.apache.zeppelin.integration.ParagraphActionsIT)  Time elapsed: 9.425 sec  <<< FAILURE!
java.lang.AssertionError: After Remove : Number of paragraphs are
Expected: <2>
     but: was <1>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.rules.ErrorCollector$1.call(ErrorCollector.java:65)
	at org.junit.rules.ErrorCollector.checkSucceeds(ErrorCollector.java:78)
	at org.junit.rules.ErrorCollector.checkThat(ErrorCollector.java:63)
	at org.apache.zeppelin.integration.ParagraphActionsIT.testRemoveButton(ParagraphActionsIT.java:156)


Results :

Failed tests: 
  ParagraphActionsIT.testRemoveButton:156 After Remove : Number of paragraphs are
Expected: <2>
     but: was <1>

Tests run: 14, Failures: 1, Errors: 0, Skipped: 0